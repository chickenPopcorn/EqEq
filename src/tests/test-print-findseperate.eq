Foo1 = {
  a = { 1; }
}

Foo1: find a {
  print("previous a = %.0f\n", a);
  a = 3;
  print("after a = %.0f\n", a);
}

Foo1: find a {
  print("previous a = %.0f\n", a);
  a = 5;
  print("after a = %.0f\n", a);
}
