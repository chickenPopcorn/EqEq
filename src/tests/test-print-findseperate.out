previous a = 1
after a = 3
previous a = 1
after a = 5
